src/Symfony/Component/AssetMapper/ImportMap/ImportMapConfigReader.php

```diff
@@ -64,7 +64,7 @@ public function getEntries(): ImportMapEntries
 
             $version = $data['version'] ?? null;
 
-            if (null === $version && null === $path) {
+            if (null === $version) {
                 throw new RuntimeException(sprintf('The importmap entry "%s" must have either a "path" or "version" option.', $importName));
```

Comment:
Is the error message still correct?

Reply:
Yes, the path is checked before so there we know data['path'] is null / empty. 

```php
 if (isset($data['path'])) {
    if (isset($data['version'])) {
        throw new RuntimeException(sprintf('The importmap entry "%s" cannot have both a "path" and "version" option.', $importName));
    }
    if (isset($data['package_specifier'])) {
        throw new RuntimeException(sprintf('The importmap entry "%s" cannot have both a "path" and "package_specifier" option.', $importName));
    }
    $entries->add(ImportMapEntry::createLocal($importName, $type, $data['path'], $isEntrypoint));
    continue;
}
```